static/js/components/forms/CouponForm_test.js

```diff
@@ -116,6 +131,29 @@ describe("CouponForm", () => {
     })
   })
 
+  //
+  ;[
+    ["activation_date", 0, "06/27/2019", "2019-06-27T00:00:00.000Z"],
```

Comment:
I think we should be submitting this formatted datetime in the browser's timezone, or have it in the UTC timezone but have midnight be in the browser's timezone, so it would be a different hour here.

Reply:
I just replaced `setUTCHours` with `setHours`, on my local instance this saves the `CouponPaymentVersion` datetime to `04:00:00` on the server.

Reply:
Is that the behavior you expect?

Reply:
Yes I think that makes sense.